### PR TITLE
Command to create Staking & Networking keys for internal nodes

### PR DESCRIPTION
**Pull request by danuio**
Resolves https://github.com/dapperlabs/flow-go/issues/4954.

### CODE REVIEW

**Comment by zhangchiqing** [comment]
Nice work!

Could you add two additional checks:
### 1. make sure the output folder is empty
If the output folder is not empty, the output folder will be mixed with old keys and new keys. 

### 2. print the summary log in the end
For instance:
```
created keys for 4 access nodes
created keys for 3 consensus nodes
created keys for 5 collection nodes
created keys for 2 execution nodes
created keys for 6 verification nodes
```

File: cmd/bootstrap/cmd/keygen.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+		}
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(keygenCmd)
+
+	// required parameters
+	keygenCmd.Flags().
+		StringVar(&flagConfig, "config", "", "path to a JSON file containing multiple node configurations (Role, Address, Stake)")
```
Comment (zhangchiqing) [inline cmd/bootstrap/cmd/keygen.go:58]:
```suggestion
		StringVar(&flagConfig, "config", "--node-config.json", "path to a JSON file containing multiple node configurations (Role, Address, Stake)")
```

File: cmd/bootstrap/cmd/keygen.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	_ = keygenCmd.MarkFlagRequired("config")
+
+	// optional parameters
+	keygenCmd.Flags().
+		BoolVar(&flagFastKG, "fast-kg", false, "use fast (centralized) random beacon key generation instead of DKG")
+}
```
Comment (zhangchiqing) [inline cmd/bootstrap/cmd/keygen.go:64]:
```suggestion
```

The keygen does not generate random beacon key, so we could remove this option

**Review by zhangchiqing — APPROVED** [review]
👍 Looks good

**Review by tarakby — APPROVED** [review]
👌

**Pull request merged**